loopchain/blockchain/vote.py

```diff
@@ -69,39 +66,38 @@ def votes(self):
     def target_hash(self):
         return self.__target_hash
 
-    @staticmethod
-    def __make_vote_init(audience):
+    def __make_vote_init(self, audience):
         if not audience:
             return None
 
         vote_init = {}
```

Comment:
This variable is initialized early. I recommend to place it where it uses.
Or just return the dict
```python
logging.debug(f"vote_init: {audience}")  # It is not need to print `vote_init`
return {peer_id: None for peer_id in audience}
```

Reply:
👍